pushapkscript/apk.py

```diff
@@ -57,16 +62,18 @@ def _extract_architecture_from_paths(apk_path, paths):
     return unique_architectures[0]
 
 
-def _check_architectures_are_valid(mozapkpublisher_architectures):
+def _check_architectures_are_valid(mozapkpublisher_architectures, channel):
+    expected_architectures = _EXPECTED_MOZAPKPUBLISHER_ARCHITECTURES_PER_CHANNEL[channel]
```

Comment:
We may hit an exception here if channel isn't in `_EXPECTED_MOZAPKPUBLISHER_ARCHITECTURES_PER_CHANNEL`.  We may want to catch `KeyError` and raise a `TaskVerificationError`.